projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-headline/order-detail-headline.component.html

```diff
@@ -1,18 +1,18 @@
 <ng-container *ngIf="order$ | async as order">
   <div class="cx-header row">
-    <div class="cx-detail col-sm-12 col-md-4">
+    <div class="cx-detail col-sm-12 col-md-3">
```

Comment:
Changing a class on an existing element is technically a breaking change

Reply:
I will revert it back.